app/assets/stylesheets/views/_topics.scss

```diff
@@ -123,7 +123,7 @@
   .browse-container {
     background-color: $page-colour;
     min-height: 500px;
-    padding: 0 30px 45px 30px;
+    padding: 0 0 45px 0;
```

Comment:
Nit: We should use long-hand where possible so it's clear what's happening here.

This is the equivalent of an a function with many arguments, whereas the long hand is easy to understand at a glance.

http://csswizardry.com/2016/12/css-shorthand-syntax-considered-an-anti-pattern/